src/Engine/Hymn.cpp

```diff
@@ -161,6 +166,10 @@ void ActiveHymn::Update(float deltaTime) {
         Managers().scriptManager->Update(world, deltaTime);
     }
     
+    { PROFILE("Update VR devices");
+        Managers().vrManager->Update();
+    }
+    
     { PROFILE("Update physics");
         Managers().physicsManager->Update(deltaTime);
     }
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Variable 'model' is assigned a value that is never used. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunreadVariable)